Audio-Ingest-Outgest/Main.cpp

```diff
@@ -0,0 +1,73 @@
+/*******************************************************
+*    This is an audio ingest/outgest station with      *
+*    full sample rate selection, along with an input   *
+*    visualizer biased towards a brown noise spectrum. *
+*******************************************************/
+
+#include "../JuceLibraryCode/JuceHeader.h"
+#include "MainComponent.h"
+
+class AudioIngestOutgestApplication  : public JUCEApplication
+{
+public:
+    AudioIngestOutgestApplication() {}
+
+    const String getApplicationName() override       { return ProjectInfo::projectName; }
+    const String getApplicationVersion() override    { return ProjectInfo::versionString; }
+    bool moreThanOneInstanceAllowed() override       { return true; }
+
+    void initialise (const String& commandLine) override
+    {
+        mainWindow.reset (new MainWindow (getApplicationName()));
+    }
+
+    void shutdown() override
+    {
+        // Quick and dirty shutdown procedure
+        mainWindow = nullptr;
+    }
+
+    void systemRequestedQuit() override
+    {
+        //Making sure OS can close out our program if needed
+        quit();
+    }
+
+    void anotherInstanceStarted (const String& commandLine) override
+    {
+
+    }
+
+    class MainWindow    : public DocumentWindow {
+    public:
+        MainWindow (String name)  : DocumentWindow 
+        (name,Desktop::getInstance().getDefaultLookAndFeel()
+        .findColour (ResizableWindow::backgroundColourId),
+        DocumentWindow::allButtons) {
```

Comment:
This is giving me anxiety just reading it. 
```suggestion
         MainWindow(String name)  : DocumentWindow(
             name,
             Desktop::getInstance().getDefaultLookAndFeel().findColour(ResizableWindow::backgroundColourId),
             DocumentWindow::allButtons
         ) {
```
Something like this might help? The old standard was 80 characters per line, but 120 has become the relatively well accepted standard, outside of old fuddy-duddies.